hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopDirTreeGenerator.java

```diff
@@ -39,7 +39,7 @@
     aliases = "dfs-tree-generator",
     description =
         "Create nested directories and create given number of files in each " +
-                "dir in any dfs compatible file system.",
+            "dir in any dfs compatible file system.",
```

Comment:
next time, let's not change the style of the unrelated code.

Reply:
Sorry about that, I have reverted the change !!